docker/docker-compose-hq.yml

```diff
@@ -10,6 +10,7 @@ web:
     - "8000:8000"
   environment:
     PYTHONUSERBASE: /vendor
+    CUSTOMSETTINGS: "docker.localsettings-docker"  
```

Comment:
This shouldn't be here otherwise it will always use these localsettings. These settings are just supposed to be for bootstrap and as an example. 

Once you've got things setup you should create your own `localsettings.py` file in the project root which will get used in future.